### PR TITLE
update arguments to reflect script changes using email engine

### PR DESCRIPTION
**Pull request by gidsg**
https://trello.com/c/3hLue99t/919-1-finish-creating-a-system-for-bulk-email-scripts-that-is-reliable-repeatable-and-auditable

Update arguments to reflect changes from https://github.com/alphagov/digitalmarketplace-scripts/pull/660.

### CODE REVIEW

**Review by alex9smith — APPROVED** [review]

**Comment by gidsg** [comment]
There is a problem whereby the log file appears to get swallowed inside the docker image, we will need to do some re-work to mount it on the host machine.

**Comment by gidsg** [comment]
@alex9smith could you re-review with the later commits?

File: job_definitions/notify_suppliers_of_awarded_briefs.yml [hunk 1]
```diff
@@ -37,16 +41,22 @@
...
+          else
+            FLAGS="$FLAGS --reference=${JOB_NAME}_${BUILD_NUMBER}"
           fi
 
           if [ "$DRY_RUN" = "true" ]; then
             FLAGS="$FLAGS --dry-run"
           fi
 
-          docker run --rm -e DM_DATA_API_TOKEN_{{ environment|upper }} digitalmarketplace/scripts scripts/notify-suppliers-of-awarded-briefs.py '{{ environment }}' $NOTIFY_API_TOKEN {{ NOTIFY_TEMPLATE_ID }} $FLAGS --verbose
+          docker run --mount type=bind,source="$(pwd)",target=/tmp --rm -e DM_DATA_API_TOKEN_{{ environment|upper }} digitalmarketplace/scripts scripts/notify-suppliers-of-awarded-briefs.py --stage '{{ environment }}' --notify-api-key $NOTIFY_API_TOKEN --notify-template-id {{ NOTIFY_TEMPLATE_ID }} $FLAGS --verbose
```
Comment (alex9smith) [inline job_definitions/notify_suppliers_of_awarded_briefs.yml:61]: Is this binding to the magic Jenkins job workspace directory which gets cleaned up every so often?

**Comment by alex9smith** [comment]
Not sure how to refresh my old review, but the new changes look fine to me 🚀

File: job_definitions/notify_suppliers_of_awarded_briefs.yml (hunk 1 above)
Comment (gidsg) [reply job_definitions/notify_suppliers_of_awarded_briefs.yml:61]: @alex9smith yes, `$(pwd)` in this instance is the Jenkins workspace directory which is where the script is run from and persists after the test run.

**Pull request merged**